src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -304,7 +304,13 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::visualise( const IECore::I
 	else
 	{
 		// Treat everything else as a point light.
-		ornaments->addChild( const_pointer_cast<IECoreGL::Renderable>( pointRays() ) );
+		const float radius = parameter<float>( metadataTarget, shaderParameters, "radiusParameter", -1 );
+		if( radius > 0 )
+		{
+			geometry->addChild( const_pointer_cast<IECoreGL::Renderable>( pointShape( radius ) ) );
```

Comment:
When the radius is small, the circle isn't visible because the colour indicator is drawn on top. Would it be better to add this _after_ the colour indicator, so it is visible on top?

Reply:
I've tried to address that one in 6752471. I couldn't see anything this breaks, but it's a bit of a weird fix that I don't fully understand.